Don't adjust guest tsc to make it match the value the tsc should have
had when a timer interrupt was supposed to be delivered.

diff -r 44a5c7fcadec xen/arch/x86/hvm/hvm.c
--- a/xen/arch/x86/hvm/hvm.c
+++ b/xen/arch/x86/hvm/hvm.c
@@ -542,6 +542,8 @@ int hvm_domain_initialise(struct domain 
 
     d->arch.hvm_domain.params[HVM_PARAM_BUFIOREQ_EVTCHN] = -1;
     d->arch.hvm_domain.params[HVM_PARAM_HPET_ENABLED] = 1;
+    d->arch.hvm_domain.params[HVM_PARAM_TIMER_MODE]
+        = HVMPTM_no_delay_for_missed_ticks;
     d->arch.hvm_domain.params[HVM_PARAM_TRIPLE_FAULT_REASON] = SHUTDOWN_reboot;
 
     hvm_init_cacheattr_region_list(d);
